packages/thumbprint-react/components/Calendar/test.tsx

```diff
@@ -31,17 +26,21 @@ describe('calls `onChange` with the correct date', () => {
             <Calendar onChange={onChange} onMonthChange={onMonthChange} value={value} />,
         );
 
-        const allDays = wrapper.find('.DayPicker-Day');
+        const allDays = wrapper.find('.rdp-day');
 
         // Click on the 15th first.
-        const day15 = allDays.findWhere(n => n.prop('children') === 15);
-        day15.simulate('click');
+        const day15 = allDays
+            .findWhere(n => {
+                return n.text() === '15';
+            })
+            .at(0);
 
+        day15.simulate('click');
         expect(onChange).toHaveBeenCalledTimes(1);
         expect(onChange).toHaveBeenCalledWith([new Date('2057-09-15T07:00:00.000Z')]);
 
         // Click on the 22nd next.
-        const day22 = allDays.findWhere(n => n.prop('children') === 22);
+        const day22 = allDays.findWhere(n => n.text() === '22').at(0);
```

Comment:
Do you know why `.at()` and `.hostNodes()` is suddenly needed?

Reply:
I think because we are getting multiple nodes with same class. So we are using hostNodes() and at() to get single node of that.

Ex: `rdp-day--theme-strikeout` class applies to `<ForwardRef>` + `<button>`. So using hostNode(), we are getting single wrapper of that. 
<img width="996" alt="Screenshot 2023-08-02 at 5 35 09 PM" src="https://github.com/thumbtack/thumbprint/assets/104916135/a32246fc-4c25-492a-9c76-dd116ca852ec">